Homeworks/05/src/components/Favorites.tsx

```diff
@@ -0,0 +1,86 @@
+import Modal from 'react-modal';
+import { Flex, Grid, styled } from '../../styled-system/jsx';
+import FavoriteCard from './components/FavoriteCard';
+import { Pokemon } from '../interface';
+import { useEffect, useState } from 'react';
+
+const arrowUrl: string = "src/images/left-arrow.png"
+
+Modal.setAppElement("#root")
+
+function Favorites({isOpen=false, onClose, pokemons=[], addFavorite, removeFavorite, theme="base"}: 
+    {isOpen: boolean, onClose: () => void, pokemons: Pokemon[], addFavorite: Function, removeFavorite: Function, theme: string}) {
```

Comment:
`Function` type je hacky, zasto ne definirati preciznije

```suggestion
    {isOpen: boolean, onClose: () => void, pokemons: Pokemon[], addFavorite: (p: Pokemon) => void, removeFavorite: (name: string) => void, theme: string}) {
```